lib/publiccloud/instance.pm

```diff
@@ -156,7 +156,14 @@ sub _prepare_ssh_cmd {
 
 =head2 ssh_script_run
 
-    ssh_script_run($cmd [, timeout => $timeout] [, fail_message => $fail_message] [,quiet => $quiet] [,ssh_opts => $ssh_opts] [,username => $username])
+    ssh_script_run($cmd [, timeout => $timeout] [,quiet => $quiet] [,ssh_opts => $ssh_opts] [,username => $username][, continue_after_timeout => $continue_after_timeout])
+
+    C<timeout> - TTL for command execution measured in seconds . After that period of time execution will be aborded
+    C<quiet> - avoid recording serial_results ( value bypass to script_run call)
+    C<ssh_opts> - ability to define additional ssh options used during ssh call
+    C<username> - username used for ssh tunnel
+    C<continue_after_timeout> - in case waiting longer than timeout normally script_run will die. Setting this parameter to true
+        will avoid such failure and allow to proceed with execution
```

Comment:
I'm still not sure what this means.

Reply:
> I meant that further execution of the test will continue , but function would abort command execution after timeout . I know it has confusing name and description but I failed to come up with better one . I am open for your suggestions for both ( actual parameter name and doc for it )

Given that we're ignoring a timeout failure, what about `ignore_timeout_failure` ?